requirements.txt

```diff
@@ -1,3 +1,4 @@
+bluepy==1.0.5
```

Comment:
The code doesn't run, in spite of installing: 
$ sudo apt-get install libglib2.0-dev

I'm getting the following error, currently working on the branch edge-intelligence:
ImportError: No module named sensortag